packages/runner-ct/src/main.jsx

```diff
@@ -13,9 +13,10 @@ configure({ enforceActions: 'always' })
 
 const Runner = {
   start (el, base64Config) {
-    action('started', () => {
+    runInAction('started', () => {
```

Comment:
Why do we need `runInAction` here? Is it because we are using `<React.Suspense>` which is internally `async`?

Reply:
Its needed because we update the mobx state inside this function. (it was here before)

Reply:
So if we have *anything* marked with `@action` in mobx, when we call that function, we must do `runInAction`? What does `runInAction` do internally (something to do with a re-render?) I guess this is similar to `dispatch(actionCreator())` in redux 🤔

Reply:
@action is decorator
Which basically doing

```js
@action 
method() {
} 

=====

method() {
   runInAction(() => {
      ...method body
   })
}
```

`action` is just a function. `runInAction` is equivalent of 

```js
action('name', () => {
  ...
})() // runItRightNow
```

Reply:
Sure, but in the mobx store we do

```
  @action updateDimensions (width, height) {
    this.width = width
    this.height = height
  }
```

so we should not need runInAction, right? since we already use `@action`

Reply:
Yes. We can remove it then ¯\_(ツ)_/¯